### PR TITLE
Upstream: Pull in changes from Hugo v0.75.0

### PR DESCRIPTION
**Pull request by chayev**
Closes: #85

### CODE REVIEW

**Comment by felicianotech** [comment]
Why is the branch name `hugo-v0.75.0-1`?

File: src/go.mod [hunk 1]
```diff
@@ -5,68 +5,63 @@ require (
 	github.com/BurntSushi/toml v0.3.1
```
Comment (felicianotech) [inline src/go.mod:5]: You never ran `go mod tidy`.

File: src/magefile.go [hunk 2]
```diff
@@ -165,13 +176,13 @@ func Test386() error {
...
 func Test() error {
 	env := map[string]string{"GOFLAGS": testGoFlags()}
-	return runCmd(env, goexe, "test", "./...", "-tags", buildTags())
+	return runCmd(env, goexe, "test", "./...", buildFlags(), "-tags", buildTags())
 }
 
 // Run tests with race detector
 func TestRace() error {
 	env := map[string]string{"GOFLAGS": testGoFlags()}
-	return runCmd(env, goexe, "test", "-p 1", "-race", "./...", "-tags", buildTags())
```
Comment (felicianotech) [inline src/magefile.go:174]: We added this change specifically for Gotham. We shouldn't remove it.

**Review by felicianotech — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# See https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#package-ecosystem
+version: 2
+updates:
+  - package-ecosystem: "gomod" 
+    directory: "/"
```
[the remark this answers is not in the document]
Comment (chayev) [reply .github/dependabot.yml:5]: updating to `/src`
good catch

File: src/magefile.go (hunk 2 above)
Comment (chayev) [reply src/magefile.go:174]: re-adding "-p 1"

File: src/go.mod (hunk 1 above)
Comment (chayev) [reply src/go.mod:5]: Correct - I was waiting for the tests to pass. Doing that now.

**Review by felicianotech — APPROVED** [review]
LGTM

**Pull request merged**